frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick-start-devperspective.feature

```diff
@@ -118,3 +114,13 @@ Feature: Quick Starts card in developer console
              When user clicks "Get started with a sample application" Quick Start
               And user drags the Quick Start drawer left and right from the left side of the panel
              Then user is able to resize the Quick Start drawer
+
+
+        @odc-5010 @manual
+        Scenario: Disabling Quick Start: QS-03-TC11
+            Given user is at Quick Start card page
+              And Exploring Serverless applications quick start is present
+              And user is at developer console in inspect element
+             When user enters 'window.SERVER_FLAGS.quickStarts = {"disabled":["serverless-application"]}' in console
+              And user clicks Enter button in keyboard
+             Then user is not able to see the Exploring Serverless applications quick start
```

Comment:
Can you please move the Given statements from scenarios to Background. We can start the scenarios with When statements as well.

Reply:
This particular statement is present in some of the last scenarios only